### PR TITLE
Update checkstyle and change tab width to be 4 than 8

### PR DESCRIPTION
**Pull request by liach**
gradlew is already executable 100755, so i only updated gradlew bat

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Comment by modmuss50** [comment]
the bat file doesnt need +x? this doesnt really change a lot.

**Comment by liach** [comment]
Oh I also added `-i` flag so gradle messages will be very informative. setting the bat +x is like just done by chance, is that not needed? (i am not sure)

However the `chmod +x gradlew` definitely can be removed from travis yml.

**Comment by modmuss50** [comment]
-i isnt needed, its just too much info on an already large build log.

**Comment by liach** [comment]
Done. Now removes the extraneous `chmod +x gradlew` line

**Comment by liach** [comment]
Now I've updated checkstyle to treat tabs as 4 width.

**Comment by liach** [comment]
Fyi that `()` removal around `world` is a checkstyle bug.

**Comment by liach** [comment]
This pull request will wait for checkstyle 5.31 release, which is [scheduled for Mar 29th](https://github.com/checkstyle/checkstyle/milestone/67), and update.

**Comment by liach** [comment]
Updated checkstyle and fixed a few bad lambda indents in the old code

**Review by modmuss50 — APPROVED** [review]

**Review by Player3324 — APPROVED** [review]

**Comment by dblsaiko** [comment]
Any reason for 8 continuation indent instead of 4?

File: checkstyle.xml [hunk 1]
```diff
@@ -107,11 +108,11 @@
 		</module>
 
 		<module name="Indentation">
-			<property name="basicOffset" value="8"/>
+			<property name="basicOffset" value="4"/>
 			<property name="caseIndent" value="0"/>
```
Comment (liach) [inline checkstyle.xml:112]: Hmm, @sfPlayer1 can I request to change this into 4 (1 tab)? It looks weird currently, like https://github.com/FabricMC/MappingPoet/blob/8dc2fd9864a2085784090ea5a9423295e64bd6c3/src/main/java/net/fabricmc/mappingpoet/Signatures.java#L210-L232

Comment (liach) [reply checkstyle.xml:112]: Will keep this.

**Pull request merged**